apps/native-component-list/src/screens/UI/SwiftUIContainerScreen.tsx

```diff
@@ -0,0 +1,106 @@
+import * as SwiftUI from '@expo/ui/components/SwiftUI';
+import { useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function SwiftUIContainerScreen() {
+  const [playSounds, setPlaySounds] = useState(true);
+  const [sendReadReceipts, setSendReadReceipts] = useState(false);
+
+  const notifyOptions = ['Direct Messages', 'Mentions', 'Anything'];
+  const [selectedNotifyIndex, setSelectedNotifyIndex] = useState<number>(0);
+  const profileImageSizes = ['Large', 'Medium', 'Small'];
+  const [selectedProfileImageSizeIndex, setSelectedProfileImageSizeIndex] = useState<number>(0);
+
+  return (
+    <View style={styles.container}>
+      <SwiftUI.Container style={styles.container}>
```

Comment:
Do we need to have this component? Ideally we'd just render all Form children that are SwiftUI native components as direct SwiftUI -> SwiftUI hierarchy. This doesn't really do anything from the user's perspective (maybe for now?)

Reply:
yes, it's necessary. now all swiftui components are pure swiftui views without any uikit views. the `SwiftUI.Container` is like the only `UIHostingController` to embed swiftui views in react-native's uikit views.